firefox_desktop/views/serp_impression.view.lkml

```diff
@@ -0,0 +1,279 @@
+  dimension: event_timestamp {
+    hidden: yes
+  }
+  dimension: experiments {
+    hidden: yes
+    sql: ${TABLE}.experiments ;;
+  }
+  dimension: glean_client_id {
+    hidden: yes
+    sql: ${TABLE}.glean_client_id ;;
+  }
+  dimension: has_ads_loaded {
+    type: yesno
+    sql: ${TABLE}.has_ads_loaded ;;
+  }
+  dimension: impression_id {
+    hidden: yes
+  }
+  dimension: is_ad_component {
+    type: yesno
+    sql: ${TABLE}.is_ad_component ;;
+  }
+  dimension: is_engaged {
+    type: yesno
+    sql: ${TABLE}.is_engaged ;;
+  }
+  dimension: is_shopping_page {
+    type: yesno
+    sql: ${TABLE}.is_shopping_page ;;
+  }
+  dimension: is_tagged {
+    type: yesno
+    sql: ${TABLE}.is_tagged ;;
+  }
+  dimension: legacy_telemetry_client_id {
+    hidden: yes
+  }
+  dimension: normalized_channel {
+    type: string
+    sql: ${TABLE}.normalized_channel ;;
+  }
+  dimension: normalized_country_code {
+    type: string
+    sql: ${TABLE}.normalized_country_code ;;
+  }
+  dimension: num_ads_hidden_reported {
+    hidden: yes
+  }
+  dimension: num_ads_loaded_reported {
+    hidden: yes
+  }
+  dimension: num_ads_notshowing {
+    hidden: yes
+  }
+  dimension: num_ads_showing {
+    hidden: yes
+  }
+  dimension: num_ads_visible_reported {
+    hidden: yes
+  }
+  dimension: num_clicks {
+    hidden: yes
+  }
+  dimension: num_expands {
+    hidden: yes
+  }
+  dimension: num_submits {
+    hidden: yes
+  }
+  dimension: os {
+    hidden: yes
+  }
+  dimension: ping_seq {
+    hidden: yes
+  }
+  dimension: sample_id {
+    type: number
+    sql: ${TABLE}.sample_id ;;
+  }
+  dimension: sap_source {
+    type: string
+    sql: ${TABLE}.sap_source ;;
+  }
+  dimension: search_engine {
+    type: string
+    sql: ${TABLE}.search_engine ;;
+  }
+  dimension_group: submission {
+    type: time
+    timeframes: [raw, date, week, month, quarter, year]
+    convert_tz: no
+    datatype: date
+    sql: ${TABLE}.submission_date ;;
+  }
+  measure: serp_impressions_count {
+    group_label: "SERP Impression Metrics"
+    description: "The number of distinct SERP Impressions "
+    type: count_distinct
+    sql: ${impression_id};;
+  }
+
+
+
+
+
+  measure: ad_impressions_count {
+    group_label: "Ad Impression Metrics"
+    description: "The number of distinct ad impressions if ads were visible"
+    type: number
+    sql: COUNT(DISTINCT IF(${num_ads_visible_reported} > 0, ${impression_id}, NULL));;
+
+  }
+
+  measure: ads_loaded{
+    group_label: "Ad Impression Metrics"
+    description: "Total number of ads loaded"
+    type: sum
+    sql: ${TABLE}.num_ads_loaded_reported;;
+
+  }
+
+  measure: ads_visible{
+    group_label: "Ad Impression Metrics"
+    description: "Total number of ads visible"
+    type: sum
+    sql: ${TABLE}.num_ads_visible_reported;;
+
+  }
+
+  measure: ads_hidden{
+    group_label: "Ad Impression Metrics"
+    description: "Total number of ads hidden"
+    type: sum
+    sql: ${TABLE}.num_ads_hidden_reported ;;
+
+  }
```

Comment:
Measures based on `num_ads_hidden_reported` can be dropped and we can use `ads_not_showing` instead.